src/web/mjs/engine/Settings.mjs

```diff
@@ -315,4 +336,40 @@ export default class Settings {
 
         }
     }
+
+    /**
+     * HYPOFLEX
+     * Save the given value for the given key in the bookmark storage
+     */
+    saveBookmarks( key, value, indentation ) {
+        return new Promise( ( resolve, reject ) => {
+            this.fs.writeFile( this.bookmarkDirectory.value + '\\hakuneko.' + key, JSON.stringify( value, undefined, indentation ), function( error ) {
+                if( error ) {
+                    reject( error );
+                } else {
+                    resolve();
+                }
+            } );
+        } );
+    }
+
+    /**
+     * HYPOFLEX
+     * Load the value for the given key from the bookmark storage
+     */
+    async loadBookmarks( key ) {
+        //return fetch( this.config + key ).then( response => response.json() );
+        return new Promise( ( resolve, reject ) => {
+            this.fs.readFile( this.bookmarkDirectory.value + '\\hakuneko.' + key, 'utf8', ( error, data ) => {
```

Comment:
Will `\\` path separator work on linux and macOS?

Reply:
i knew this would maybe cause an issue
have not tested on linux/osx yet, didnt have my laptop in reach, will test later